src/js/edu-benefits/1990e/config/form.js

```diff
@@ -111,6 +118,55 @@ const formConfig = {
     sponsorVeteran: {
       title: 'Sponsor Veteran',
       pages: {
+        sponsorVeteran: {
+          title: 'Sponsor Veteran',
+          path: 'sponsor-veteran',
+          uiSchema: {
+            sponsorVeteran: {
+              veteranFullName: {
+                first: {
+                  'ui:title': 'Veteran first name'
+                },
+                last: {
+                  'ui:title': 'Veteran last name'
+                },
+                middle: {
+                  'ui:title': 'Veteran middle name'
+                },
+                suffix: {
+                  'ui:title': 'Veteran suffix',
+                  'ui:options': {
+                    widgetClassNames: 'form-select-medium'
+                  }
+                }
+              },
+              veteranSocialSecurityNumber: _.set(ssnCommon.uiSchema, ['ui:title'], 'Veteran Social Security number'),
```

Comment:
@annekainicUSDS I think you should switch this to import `lodash/fp` at the top of the file. That's what's causing that problem, I think. The regular lodash set modifies the common ssn uiSchema.

You'll have to move the `ssnCommon.uiSchema` argument to the end of the function when you do that.